src/main/java/me/moodcat/database/entities/ChatMessage.java

```diff
@@ -60,9 +60,9 @@
      *            The author to set.
      * @return The author that made this chatmessage
      */
-    @Column(name = "author", nullable = false)
-    @NonNull
-    private String author;
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "user_id", nullable = false)
```

Comment:
id is used for user id, camelcase for column names is not preferable becuase the Postgres JDBC driver sucks with camelcase column names